src/mixins/audio_token_behavior.mixin.js

```diff
@@ -0,0 +1,81 @@
+(function() {
+
+  fabric.util.object.extend(fabric.Audio_token.prototype, /** @lends fabric.Audio_token.prototype */ {
+
+    /**
+     * Initializes all the interactive behavior of Audio_token
+     */
+    initBehavior: function() {
+      this.initAddedHandler();
+      this.initRemovedHandler();
+      this.initMousedownHandler();
+    },
+
```

Comment:
I'm intentionally keeping the console.logs here for demonstrating the behavior in amp-worksheet.

Reply:
How does fabric deal with logging? 

Would it be useful to be able to enable/disable this set of logs around audio tokens for future debugging?

Reply:
There is a fabric.log set = console.log but it isnt used much at all. I'll change these to fabric.log then we will be able to turn it on/off from within amp-worksheet if we want.

Reply:
Might want a local flag to enable just for this feature vs the rest to avoid getting spammed when you want to just see these. Think classroom has this pattern.